### ISSUE TITLE
The name, role, state, and value of all user interface components can be programmatically determined (especially when the state changes).

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#150]
Screen-reader users do not receive feedback as to the state of the pause/play slide button in the hero carousel. Apply aria-live="polite" to the .sr-only span that labels the button.

Screen-reader users do not receive feedback when the slide count changes in the hero carousel. We recommend wrapping the "X/Y" count in a container and apply role="status".

The tabs spanning "Explore Programs" to "Research Areas" lack proper semantics and keyboard functionality for screen-reader users. We recommend reviewing the WAI-ARIA [Tabs with Manual Activation](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/) pattern for more details.
